scalyr_agent/tests/syslog_monitor_test.py

```diff
@@ -257,6 +259,22 @@ def test_config_port_too_high(self):
         )
 
 
+class TestAutoFlushingRotatingFileHandler(AutoFlushingRotatingFileHandler):
+    """subclass of AutoFlushingRotatingFileHandler with overridden '_internal_flush'
+    to notify test thread that stream was flushed.
+    AutoFlushingRotatingFileHandler is inherited from old-style class in python2.6, that's why we can not use 'super'.
+    """
+
+    def __init__(self, *args, **kwargs):
+        AutoFlushingRotatingFileHandler.__init__(self, *args, **kwargs)
```

Comment:
``super(AutoFlushingRotatingFileHandler, self).__init__(..)`` is usually more robust  and preferred in case the inheritance hierarchy changes.

Reply:
Unfortunately in python2.6 `AutoFlushingRotatingFileHandler` belongs to a class hierarchy that begins from an old-style class :( 
Sorry, I had to put this information in constructor docstring, not a class.